micromasters/settings.py

```diff
@@ -493,6 +493,14 @@ def get_var(name, default):
         'task': 'financialaid.tasks.sync_currency_exchange_rates',
         'schedule': crontab(minute=0, hour='3')
     },
+    'export_exam_profiles-every-1-hrs': {
+        'task': 'exams.tasks.export_exam_profiles',
+        'schedule': crontab(minute=0, hour='1')
```

Comment:
This says "run every day at 1am". You need `crontab(minute=0, hour='*')` to run every hour.